src/NHibernate/Cfg/MappingSchema/Hbm.generated.cs

```diff
@@ -2,7 +2,7 @@ namespace NHibernate.Cfg.MappingSchema {
     
     
     /// <remarks/>
-    [System.CodeDom.Compiler.GeneratedCodeAttribute("HbmXsd", "5.0.0.Alpha1")]
+    [System.CodeDom.Compiler.GeneratedCodeAttribute("HbmXsd", "5.0.0-Alpha1")]
```

Comment:
Mass change due to the NHibernate versioning change.

Going to squash the fixup before TC starts building.